lib/pcre.mli

```diff
@@ -29,6 +29,12 @@ val exec : rex:regexp -> ?pos:int -> string -> groups
 val get_substring : groups -> int -> string
 (** Equivalent to {!Core.Group.get}. *)
 
+val names : regexp -> string array
+(** Return the names of named groups. *)
+
+val get_named_substring : regexp -> string -> groups -> string
```

Comment:
Named arguments are convenient for function call dyslexics like me!

Reply:
Maybe, but since this is supposed to be a compatibility layer, I kept the same API as the original :-)

Reply:
Word makes sense.